src/groups/z_bmq/z_bmqa/z_bmqa_confirmeventbuilder.h

```diff
@@ -0,0 +1,34 @@
+#ifndef INCLUDED_Z_BMQA_CONFIRMEVENTBUILDER
+#define INCLUDED_Z_BMQA_CONFIRMEVENTBUILDER
+
+#include <z_bmqt_types.h>
+#include <z_bmqa_message.h>
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+typedef struct z_bmqa_ConfirmEventBuilder z_bmqa_ConfirmEventBuilder;
+typedef struct z_bmqa_MessageConfirmationCookie z_bmqa_MessageConfirmationCookie;
+
+int z_bmqa_ConfirmEventBuilder__delete(z_bmqa_ConfirmEventBuilder** builder_obj);
```

Comment:
I don't necessarily agree that doing the double pointer trick for destruction is the way we should go, let me think about it.